includes/classes/SiteAutomation/Request.php

```diff
@@ -96,8 +97,26 @@ public function get( $page, $widget, $timeout = 3 ) {
 		 */
 		$bypass_cache = apply_filters( 'sophi_bypass_get_cache', false, $page, $widget );
 
-		if ( ! $bypass_cache ) {
-			$site_automation_data = get_option( "sophi_site_automation_data_{$page}_{$widget}" );
+		if ( ! $bypass_cache && post_type_exists('sophi-response' ) ) {
```

Comment:
Do we even have a workflow when `sophi-response` post type not exist?

```suggestion
		if ( ! $bypass_cache && post_type_exists( 'sophi-response' ) ) {
```

Reply:
Makes sense to me @cadic I have adjusted it here https://github.com/globeandmail/sophi-for-wordpress/pull/298/commits/b94450894c6af45443f335d7933334667454aeb1